src/Jackett.Common/Models/Config/RuntimeSettings.cs

```diff
@@ -37,7 +37,7 @@ public string DataFolder
 
                 if (System.Environment.OSVersion.Platform == PlatformID.Unix)
                 {
-                    return Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Jackett");
+                    return Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData, Environment.SpecialFolderOption.Create), "Jackett");
```

Comment:
@NinjaLikesCheez  Does this code apply to osx? do macs also use PlatformID.Unix ?

Did you test this on your mac, I know you tested the earlier attempt, but you have not specified if you tested for this latest code change in any comment since?
Thanks.

Reply:
@garfield69 Hi! Thanks for taking a look at this. Yes, I did test the new change on macOS - from the PlatformID Docs it appears this is the correct value to use:

```
MacOSX | 6 | The operating system is Macintosh. This value was returned by Silverlight. On .NET Core, its replacement is Unix.
```

https://docs.microsoft.com/en-us/dotnet/api/system.platformid?view=net-5.0